src/os/bluestore/aio.cc

```diff
@@ -5,6 +5,17 @@
 
 #if defined(HAVE_LIBAIO)
 
+std::ostream& operator<<(std::ostream& os, const aio_t& aio)
+{
+  unsigned i = 0;
+  os << "aio: ";
+  for (auto& iov : aio.iov) {
+    os << "\n [" << i++ << "] " << std::hex
+       << iov.iov_base << " " << iov.iov_len;
```

Comment:
please reset to decimal on completion to avoid potential side effects..

Reply:
@ifed01 fixed and repushed